app/screens/home/search/search.tsx

```diff
@@ -179,25 +235,21 @@ const SearchScreen = ({teamId}: Props) => {
                         <RoundedHeaderContext/>
                         {header}
                     </Animated.View>
-                    {!showResults &&
-                        <Modifiers
-                            setSearchValue={setSearchValue}
-                            searchValue={searchValue}
-                            scrollPaddingTop={scrollPaddingTop}
-                        />
-                    }
-                    {showResults &&
-                        <Results
-                            selectedTab={selectedTab}
-                            searchValue={lastSearchedValue}
-                            postIds={postIds}
-                            fileInfos={fileInfos}
-                            scrollRef={scrollRef}
-                            onScroll={onScroll}
-                            scrollPaddingTop={scrollPaddingTop}
-                            loading={loading}
-                        />
-                    }
+                    <AnimatedFlatList
```

Comment:
If you are rendering just one element for each possibility, why not use a scroll view instead of a flat list?

Reply:
We used the AnimatedFlat list to animate the items when tapping the `Show more` / `Show less` button.